cookbook-react/src/recipes/buttons/button-1/styles.scss

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable-next-line plugin/stylelint-no-indistinguishable-colors */
```

Comment:
disabled indistinguishable colors, is this still necessary?

Reply:
sadly yes